frontend/src/pages/admin/FlowBlueprints/dialogs/FlowBlueprintFormDialog.vue

```diff
@@ -179,17 +179,33 @@ export default {
 
             // Validation
             try {
-                flowBlueprintProps.flows = JSON.parse(flowBlueprintProps.flows)
+                JSON.parse(flowBlueprintProps.flows)
             } catch (err) {
                 this.error = 'Invalid JSON for flows'
+                this.errors.flows = 'Invalid JSON'
+                return
+            }
+
+            try {
+                if (!JSON.parse(flowBlueprintProps.flows).flows) {
+                    throw new Error('Flow json missing \'flows\' property')
+                }
+                if (!Array.isArray(JSON.parse(flowBlueprintProps.flows).flows)) {
+                    throw new Error('Flow json \'flows\' property not an Array')
+                }
+            } catch (err) {
+                this.error = err
                 this.errors.flows = err
                 return
             }
+
+            flowBlueprintProps.flows = JSON.parse(flowBlueprintProps.flows)
+
             try {
                 flowBlueprintProps.modules = JSON.parse(flowBlueprintProps.modules)
```

Comment:
Same issue as before, this validation will permit null, "", "123", 123, [] and [ {"name: "node-red-dashboard", "version": "1.9.1"}] since they are ALL valid JSON, but they will break the blueprint!

Reply:
I was lazy enough to copy over the validation from BE. Will update now

Reply:
On the long run we should transition to more robust validation libraries like https://vuelidate-next.netlify.app/ or https://vee-validate.logaretm.com/v3/guide/basics.html